encoding/int/encoding.go

```diff
@@ -0,0 +1,46 @@
+package int
+
+import "io"
+
+// Encoder encodes int values.
+type Encoder interface {
+	// Encode a collection of ints.
+	// Callers should explicitly call `Reset` before
+	// subsequent call to `Encode`.
+	Encode(Iterator) error
```

Comment:
nit: conventionally we spell out the var names in the APIs for clarity.

Reply:
👍